easel/lib/screens/creator_hub/widgets/draft_list_tile.dart

```diff
@@ -243,8 +243,8 @@ class _DraftListTileState extends State<DraftListTile> {
                   ),
                 ),
               )
-            : const IgnorePointer(
-                child: SizedBox(),
+            : IgnorePointer(
+                child: SizedBox(height: 85.h,),
```

Comment:
code formatting not done

Reply:
@ahmadrns it is done using dart formatter. it is exactly 2 spaces.